docs/topics/multiplatform-mobile/multiplatform-mobile-understand-project-structure.md

```diff
@@ -438,7 +436,6 @@ android {
     defaultConfig {
         applicationId = "org.example.androidApp"
         minSdk = 24
-        targetSdk = 29
```

Comment:
Why `targetSdk` has been removed? I see that [Android docs](https://developer.android.com/build) mention it

Reply:
IDE always complains that it's deprecated and can be removed. As far as I understand, [it was deprecated for com.android.library](https://developer.android.com/reference/tools/gradle-api/7.4/com/android/build/api/dsl/LibraryBaseFlavor#targetSdk()).